controller/tests/test_volunteer_controller.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+import controller.volunteer_controller as vc
+from models.volunteer import Volunteer
+from models.camp import Camp
+
+
+class TestVolunteerController(unittest.TestCase):
+
+    def test_create_volunteer(self):
+        volunteer_called = vc.create_volunteer(username='yunsy', password='root', firstname='Yunsy', lastname='Yin',
```

Comment:
In general in the tests, I think just naming this `volunteer` would be fine. 
`volunteer_called` doesn't make much sense in this context. 
(I wrote `volunteer_called` once because the function has 2 volunteer variables. So it is just for telling them apart)